src/main/java/frc/team2412/robot/util/AutoLogic.java

```diff
@@ -65,7 +65,7 @@ public void registerCommands() {
 						.andThen(new WaitCommand(1))
 						.andThen(new FeederInCommand(s.intakeSubsystem)));
 
-		NamedCommands.registerCommand("Intake", new AllInCommand(s.intakeSubsystem));
+		NamedCommands.registerCommand("Intake", new AllInCommand(s.intakeSubsystem, controls));
```

Comment:
Note that since this is in auto we don't need to rumble the drive controller... probably okay to leave in though.

Reply:
`AllInCommand` could be updated to tolerate `null` being passed for `controls` and skip the controller vibration in that case. That'd avoid the controllers vibrating during auto and potentially falling off the drive board.